rhobs/alerting/data_plane/prometheus.pipeline_alerts.yaml

```diff
@@ -146,3 +146,39 @@ spec:
             team: pipelines
             # link will be available once https://issues.redhat.com/browse/SRVKP-5899 is complete
             runbook_url: https://gitlab.cee.redhat.com/konflux/docs/sop/-/blob/main/pipeline-service/slos/chains-performance.md
+
+        - alert: PACControllerDeadlock
+          expr: |
+            sum by (source_cluster) (increase(pac_watcher_work_queue_depth[2m])) > 50
+          for: 75m
+          labels:
+            severity: critical
+            slo: "true"
+          annotations:
+            summary: >-
+              Tekton PAC controller appears to have stopped processing active pipelineruns.
+            description: >-
+              Tekton PAC controller on cluster {{ $labels.source_cluster }} has appeared deadlocked on {{ $value }} pipelinerun events.
+            alert_team_handle: <!subteam^S04PYECHCCU>
+            team: pipelines
+            # link will be available once https://gitlab.cee.redhat.com/konflux/docs/sop/-/merge_requests/163 merges
+            runbook_url: https://gitlab.cee.redhat.com/konflux/docs/sop/-/blob/main/pipeline-service/slos/tekton-pipeline-related-deadlocks.md
+        - alert: PACControllerPerformance
+          expr: |
+            (sum by (source_cluster) (increase(pac_watcher_client_latency_bucket{le='1'}[5m]))
+            /
+            sum by (source_cluster) (increase(pac_watcher_client_latency_bucket{le='+Inf'}[5m])))
+            < 0.50
+          for: 75m
+          labels:
+            severity: critical
+            slo: "true"
+          annotations:
+            summary: >-
+              Tekton PAC controller performance appears degraded with uncommonly high client latency.
+            description: >-
+              Tekton PAC controller on cluster {{ $labels.source_cluster }} performance appears degraded with client latency {{ $value | humanizePercentage }}.
+            alert_team_handle: <!subteam^S04PYECHCCU>
+            team: pipelines
+            # link will be available once https://issues.redhat.com/browse/SRVKP-5899 is complete
+            runbook_url: https://gitlab.cee.redhat.com/konflux/docs/sop/-/blob/main/pipeline-service/slos/pac-performance.md # TODO: Change URL
```

Comment:
fyi there is probably a lot of overlap with chains wrt this alert ... I'll keep this mind mind as I start working on `chains-performance.md`

that said, as I'm working on these SOPs, the notion of high latency here may not be "actionable" ...related to the Aug 20 konflux arch call and my comment in https://gitlab.cee.redhat.com/konflux/docs/sop/-/merge_requests/163#note_12355622

A possible outcome is we remove this alert and the analogous chains one, at least for the time being